capellambse/model/common/element.py

```diff
@@ -277,7 +277,11 @@ def _short_repr_(self) -> str:
             mytype = f"Model element ({self.xtype})"
         else:
             mytype = type(self).__name__
-        return f"<{mytype} {self.name!r} ({self.uuid})>"
+        if self.name:
+            name = f" {self.name!r}"
+        else:
+            name = ""
+        return f"<{mytype}{name} ({self.uuid})>"
```

Comment:
You don't like the one liner here?

```suggestion
        name = f" {self.name!r}" if self.name else ""
```

Reply:
No, I don't. I think it unnecessarily obscures the actual program flow.